File: problems/basic/1/solution1.py
# Accepted: 1720ms (32.61%), 18.14MB (24.55%)

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)-1):
            for j in range(i+1, len(nums)):
                if nums[i] + nums[j] == target:
                    return [i, j]

        return [-1, -1] # failed

File: problems/1/solution1.ts
// Accepted: 88ms (40.49%), 43.60MB (93.75%)

function twoSum(nums: number[], target: number): number[] {
    for (let i = 0; i < nums.length-1; ++i) {
        for (let j = i+1; j < nums.length; ++j) {
            if (nums[i] + nums[j] === target) {
                return [i, j];
            }
        }
    }

    return [-1, -1]; // failed
};
